worker/src/RTC/SeqManager.cpp

```diff
@@ -113,15 +113,25 @@ namespace RTC
 
 		output = (input + base) & MaxValue;
 
-		// New input is higher than the maximum seen. But less than acceptable units higher.
-		// Keep it as the maximum seen. See Drop().
-		if (IsSeqHigherThan(input, this->maxInput))
-			this->maxInput = input;
-
-		// New output is higher than the maximum seen. But less than acceptable units higher.
-		// Keep it as the maximum seen. See Sync().
-		if (IsSeqHigherThan(output, this->maxOutput))
+		if (!this->started)
+		{
+			this->started = true;
+
+			this->maxInput  = input;
 			this->maxOutput = output;
+		}
+		else
+		{
+			// New input is higher than the maximum seen. But less than acceptable units higher.
+			// Keep it as the maximum seen. See Drop().
+			if (IsSeqHigherThan(input, this->maxInput))
+				this->maxInput = input;
+
+			// New output is higher than the maximum seen. But less than acceptable units higher.
+			// Keep it as the maximum seen. See Sync().
+			if (IsSeqHigherThan(output, this->maxOutput))
+				this->maxOutput = output;
```

Comment:
Please let's add { } in all if/else conditions. I'm doing it little by little (there is a option for clang-tidy but it no longer works in my system due to rules that do not exist in the clang-tidy version that in theory we should use).

Reply:
Do you know which one? we should remove it then.